playbooks/robusta_playbooks/prometheus_enrichments.py

```diff
@@ -99,6 +99,33 @@ def get_prometheus_series(prometheus_params: PrometheusGetSeriesParams) -> dict:
         raise e
 
 
+class PrometheusGetLabelNames(PrometheusParams):
+    """
+    :var match: List of Prometheus series selectors.
```

Comment:
I assume the UI will always call it without params
Can you please add in the comments, what is the behavior when leaving each of the params empty.

Reply:
@arikalon1 Added comments